src/clr-angular/forms/common/error.ts

```diff
@@ -1,16 +1,39 @@
 /**
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { Component } from '@angular/core';
+import { Component, Optional, HostBinding, Input } from '@angular/core';
+import { ControlIdService } from './providers/control-id.service';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'clr-control-error',
   template: `
     <ng-content></ng-content>
     `,
-  host: { '[class.clr-subtext]': 'true' },
+  host: {
+    '[class.clr-subtext]': 'true',
+    '[attr.aria-live]': '"polite"',
```

Comment:
Do we want this to be "assertive"?

Reply:
The issue with assertive currently is if the form validation is triggered multiple times in the form (quickly tab, force form validation programmatically) the assertive will cut off other assertive announcements since ideally there should only be one aria-live per form to handle all messages.

This is a stop-gap for messages to be announced until we can come up with a solution on how to group all of the messages into a single aria-live assertive in a single form.